webapp/channels/src/components/integrations/confirm_integration/confirm_integration.tsx

```diff
@@ -135,7 +135,7 @@ const ConfirmIntegration = ({team, location, commands, oauthApps, incomingHooks,
                     id='add_incoming_webhook.url'
                     defaultMessage='<b>URL</b>: {url}'
                     values={{
-                        url: '`' + incomingHookToken + '`',
+                        url: <span><code style={{display: 'inline'}}>{incomingHookToken}</code></span>,
```

Comment:
Are you sure that these need the outer `span` element or the `display: inline` on the `code` tag? Playing around with this locally, I think that this would be sufficient
```suggestion
                        url: <code>{incomingHookToken}</code>,
```

Reply:
You're correct. I was going off of the <pre><code> example you posted and didn't actually check if just the code tag sufficed 😅 my bad.

Reply:
I'm trying to remember where I needed `pre` tags around the `code` tags since I swore they were needed. Maybe it was for code blocks in Markdown specifically. Sorry for the confusion though